server/controllers/spreadsheet-controller.js

```diff
@@ -401,79 +422,158 @@ var errorList = {
+  68: 'BQ',
+  69: 'BR',
+  70: 'BS',
+  71: 'BT',
+  72: 'BU',
+  73: 'BV',
+  74: 'BW',
+  75: 'BX',
+  76: 'BY'
+}
 
 
 // This is the list of warnings.
 // The graph will still load if warnings are detected, but these will be reported to the user.
 
 
 var warningsList = {
-  missingSourceGeneWarning: function (row, column) {
+  missingSourceGeneWarning: function(row, column) {
     var colLetter = numbersToLetters[column];
     var rowNum = row + 1;
     return {
       warningCode: "MISSING_SOURCE",
-      errorDescription: "A source gene name is missing in cell " + colLetter+rowNum + "."
+      errorDescription: "A source gene name is missing in cell " + colLetter + rowNum + "."
     }
   },
 
-  missingTargetGeneWarning: function (row, column) {
+  missingTargetGeneWarning: function(row, column) {
     var colLetter = numbersToLetters[column];
     var rowNum = row + 1;
     return {
       warningCode: "MISSING_TARGET",
-      errorDescription: "A target gene name is missing in cell " + colLetter+rowNum + "."
+      errorDescription: "A target gene name is missing in cell " + colLetter + rowNum + "."
     }
   },
 
-  invalidMatrixDataWarning: function (row, column) {
+  invalidMatrixDataWarning: function(row, column) {
     var colLetter = numbersToLetters[column];
     var rowNum = row + 1;
     return {
       warningCode: "INVALID_DATA",
-      errorDescription: "The value in cell " + colLetter+rowNum + ", is undefined."
+      errorDescription: "The value in cell " + colLetter + rowNum + ", is undefined."
     }
   },
 
-  randomDataWarning: function (type, row, column) {
+  randomDataWarning: function(type, row, column) {
     var colLetter = numbersToLetters[column];
     var rowNum = row + 1;
     return {
       warningCode: "RANDOM_DATA",
-      errorDescription: "The value in cell " + colLetter+rowNum + ", has a corresponding source and/or target gene that is detected as " + type + "."
+      errorDescription: "The value in cell " + colLetter + rowNum + ", has a corresponding source and/or target gene that is detected as " + type + "."
     }
   },
 
-  emptyRowWarning: function (row) {
+  emptyRowWarning: function(row) {
     var rowNum = row + 1;
     return {
       warningCode: "EMPTY_ROW",
       errorDescription: "Row " + rowNum + " was found to contain no data."
     }
   },
 
-  networkSizeWarning: function (genesLength, edgesLength) {
+  networkSizeWarning: function(genesLength, edgesLength) {
     return {
       warningCode: "INVALID_NETWORK_SIZE",
       errorDescription: "Your network has " + genesLength + " genes, and " + edgesLength + " edges. Please note that networks are recommended to have less than 50 genes and 100 edges."
     }
-  }
+  },
+
+  incorrectlyNamedSheetWarning: function() {
+    return {
+      warningCode: "INCORRECTLY_NAMED_SHEET",
+      errorDescription: "The uploaded spreadsheet was named incorrectly. Please check if the sheet(s) in the uploaded spreadsheet have been named properly."
```

Comment:
I think this could be written a bit more clearly. I think something like "The uploaded file appears to contain a weighted network, but contains no 'network_optimized_weights' sheet. A weighted network must be contained in a sheet called 'network_optimized_weights' in order to be drawn as a weighted graph. Please check if the sheet(s) in the uploaded spreadsheet have been named properly."